pbe/breakupCoalescenceTables.py

```diff
@@ -0,0 +1,51 @@
+import fluid
+import numpy as np
+import matplotlib.pyplot as plt
+
+path = 'validationData/tables/1/'
+f = open(path + 'parameters', 'w')
+f_br = open(path + 'breakup', 'w')
+f_c = open(path + 'coalescence', 'w')
+
+F = fluid.fluid('coulaloglou', 0, 0)
+F.C1 = 1.52e-02
+F.C2 = 6.78e-02
+F.C3 = 1.06e-12
+F.C4 = 5.13e13
```

Comment:
I just don't like the idea of placing C1 and C2 which are breakup model constants and C3 and C4 which are coalescence model constants into one variable (or one list, one dictionary, ...), which is IMHO misleading, just to save two lines of code